installing_on_kubernetes/_topics/cluster_image_policy.md

```diff
@@ -0,0 +1,24 @@
+## Allow docker.io/manageiq images in kubernetes
```

Comment:
This section might be difficult for downstream.

Reply:
Yeah, we probably don't want this section downstream.

Reply:
is there anyway to make this consumable downstream via variable substitution?

Reply:
Downstream shouldn't be pulling from docker.io, only the already registered downstream container registry.